tools/build_api.py

```diff
@@ -260,117 +277,67 @@ def build_project(src_path, build_path, target, toolchain_name,
 
 def build_library(src_paths, build_path, target, toolchain_name,
          dependencies_paths=None, options=None, name=None, clean=False, archive=True,
-         notify=None, verbose=False, macros=None, inc_dirs=None, inc_dirs_ext=None,
+         notify=None, verbose=False, macros=None, inc_dirs=None,
```

Comment:
Why was `inc_dirs_ext` removed? Were all the callers of `build_library` updated with this change in mind?